crates/sui-framework/packages/sui-framework/sources/vec_map.move

```diff
@@ -78,6 +78,17 @@ module sui::vec_map {
         &entry.value
     }
 
+    /// Safely try borrow a value bound to `key` in `self`.
+    /// Return Some(&V) if the value exists, None otherwise.
+    /// Only works for a "copyable" value as references cannot be stored in `vector`.
+    public fun try_get<K: copy, V: copy>(self: &VecMap<K,V>, key: &K): Option<V> {
```

Comment:
We are a bit inconsistent with our naming, I think there is `get_idx` and `get_idx_opt` in this file. Not sure if this should be `get_opt`, but I see some `try_` prefix's in other spots... oh whale